### DISCUSSION TITLE
Spacy 2.x support lifetime

### DISCUSSION THREAD

**Discussion by saboehnke** [explosion/spaCy#9507] [Help: Other Questions] [answered]
Are there any links describing how long Spacy 2.x will be supported? My team and I have some implementations using 2.x and we want to make sure that we will not run into issues anytime soon if we don't upgrade for a while.

**Comment by polm** [comment, marked as answer]
We don't have an explicit end-of-life timeline for this, but our support for v2 is already somewhat limited. We aren't actively working on new releases, though we do make bugfix releases occasionally. That will probably continue until v4 comes out at some point in the future.

You are still welcome to report issues or ask questions in Discussions of course. The main difficulty with v2 systems is that if they're using older versions of Python, it may be hard for us to replicate your issues as those versions reach EOL. So partly from that perspective I'd encourage you to upgrade to v3, or to at least upgrade your Python version if you can.

If you're having trouble with migration, you can also hire us to take care of it for you. See [here](https://form.typeform.com/to/vMs2zSjM) for details.
